docker-compose.build.yml

```diff
@@ -1,6 +1,6 @@
 version: '3.3'
 
 services:
-  playground:
-    image: gitbase-playground-dev
+  web:
```

Comment:
Here and below, would `web-client` be a bit more descriptive?

Reply:
Let's call it gitbase-web, even more descriptive

Reply:
Done in 04407f3